elasticsearch/files/elasticsearch.yml

```diff
@@ -205,23 +220,18 @@ index.number_of_replicas: {{ server.get('index', {}).get('replicas', 1) }}
 # communication. (the range means that if the port is busy, it will automatically
 # try the next port).
 
-# Set the bind address specifically (IPv4 or IPv6):
-#
-#
-network.bind_host: {{ server.bind.address }}
-http.port: {{ server.bind.port }}
-http.enabled: true
-# Set the address other nodes will use to communicate with this node. If not
-# set, it is automatically derived. It must point to an actual IP address.
-#
-{%- if server.publish_host is defined %}
-network.publish_host: {{ server.publish_host }}
```

Comment:
I was just thinking of this yes. Also to keep the compatibility if someone is already using this formula and set publish_host. I will do it.